### PR TITLE
SAK-49584 Gradebook GradeType ordinal starts at 1

### PR DESCRIPTION
**Pull request by ern**
https://sakaiproject.atlassian.net/browse/SAK-49584

### CODE REVIEW

File: gradebookng/api/src/main/java/org/sakaiproject/grading/api/model/GradebookAssignment.java [hunk 1]
```diff
@@ -542,11 +542,11 @@ public String getItemType() {
                 return this.itemType;
             }
 
-            if (gb.getGradeType() == GradeType.POINTS) {
+            if (GradeType.POINTS.equals(gb.getGradeType())) {
```
Comment (ottenhoff) [inline gradebookng/api/src/main/java/org/sakaiproject/grading/api/model/GradebookAssignment.java:545]: does it make sense to set a default to points?

**Review by ottenhoff — APPROVED** [review]
LGTM. Can you also fix this broken Javadoc link?

`// These have been deprecated in favour of the {@link GradingType} enum`

**Comment by ottenhoff** [comment]
Awesome. Thank you 👍

File: gradebookng/api/src/main/java/org/sakaiproject/grading/api/model/GradebookAssignment.java (hunk 1 above)
Comment (ern) [reply gradebookng/api/src/main/java/org/sakaiproject/grading/api/model/GradebookAssignment.java:545]: Gradebook has the default set:
private GradeType gradeType = GradeType.POINTS;

Comment (ottenhoff) [reply gradebookng/api/src/main/java/org/sakaiproject/grading/api/model/GradebookAssignment.java:545]: awesome. thanks!

**Pull request merged**